projects/scion/toolkit/uuid/src/uuid.spec.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2018-2023 Swiss Federal Railways
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ *  SPDX-License-Identifier: EPL-2.0
+ */
+
+import {UUID} from './uuid.util';
+
+describe('UUID', () => {
+
+  it('should generate an UUID with 36 characters', () => {
+    const uuid = UUID.randomUUID();
+    expect(uuid).not.toBeNull();
+    expect(uuid.length === 36).toBeTruthy();
+  });
+
+  it('should generate 500000 unique UUIDs', () => {
+    // check uniqueness of 500.000 UUIDs
+    const uuids = new Set();
+    for (let i = 0; i < 500000; i++) {
+      const uuid = UUID.randomUUID();
+      expect(uuids.has(uuid)).toBeFalsy();
```

Comment:
We prefer `toBeFalse` instead of `toBeFalsy`

Reply:
Fixed